### PR TITLE
3.0 correction on addCase function

### PR DESCRIPTION
**Pull request by antograssiot**
I think that this sentence needs to be inverted.

It makes more sense with the following example and also seems to be like this in the [source code](https://github.com/cakephp/cakephp/blob/master/src/Database/Expression/CaseExpression.php#L67)

### CODE REVIEW

File: fr/orm/query-builder.rst [hunk 1]
```diff
@@ -417,7 +417,7 @@ MEDIUM, ou LARGE, nous pourrions faire ce qui suit::
     #   WHEN population >= 999001 THEN 'LARGE'
     #   END
 
-A chaque fois qu'il y moins de valeurs qu'il n'y a de conditions, ``addCase``
+A chaque fois qu'il y moins de conditions qu'il n'y a de valeurs, ``addCase``
```
Comment (HavokInspiration) [inline fr/orm/query-builder.rst]: Il manque un "a" 

"qu'il y a moins"

**Comment by HavokInspiration** [comment]
I agree

**Comment by antograssiot** [comment]
@HavokInspiration updated

**Comment by HavokInspiration** [comment]
Nice catch @antograssiot

**Pull request merged**